icu_benchmarks/models/ml_models/lgbm.py

```
@@ -47,6 +72,14 @@ def predict(self, features):
         """
         return self.model.predict_proba(features)
 
+    def _explain_model(self, reps, labels):
+        if not hasattr(self.model, "feature_importances_"):
+                raise ValueError("Model has not been fit yet. Call fit_model() before getting feature importances.")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)